pkg/store/storepb/rpc.proto

```diff
@@ -122,6 +122,9 @@ message SeriesRequest {
   // shard_info is used by the querier to request a specific
   // shard of blocks instead of entire blocks.
   ShardInfo shard_info = 13;
+
+  // Control whether the store should calculate the checksum/hash of chunks
```

Comment:
```suggestion
  // Control whether the store should calculate the checksum/hash of chunks,
```

Reply:
Out of curiosity, why the comma here?

Reply:
I think it should be `.` :smile:

Reply:
Put a dot `.` now. Makes more sense. Thx